docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -581,16 +615,54 @@ DROP TABLE <table_name>;
 The `date_based_publisher.permission_resolver` Symfony Service deprecated in v2.5 has been removed. 
 Instead, you can inject `eZ\Publish\API\Repository\PermissionResolver` and rely on auto-wiring.
 
+### Symfony MIME component
+
+The deprecated `Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface` has been replaced with `Symfony\Component\Mime\MimeTypesInterface`.
+
 ### Template parameter names
 
 The SiteAccess-aware `pagelayout` setting is deprecated in favor of `page_layout`.
 
 View parameter `pagelayout` set by `pagelayout` setting is deprecated in favor of  `page_layout`.
 
+### Controllers
+
+`eZ\Bundle\EzPublishCoreBundle\Controller` now extends `Symfony\Bundle\FrameworkBundle\Controller\AbstractController` instead of `Symfony\Bundle\FrameworkBundle\Controller\Controller` which has limited access to the dependency injection container.
```

Comment:
This change is from kernel not ezplatform-workflow, https://github.com/ezsystems/ezpublish-kernel/pull/2781/files